lib/Yam/ControlCenter.pm

```diff
@@ -0,0 +1,374 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenter;
+use parent 'YaST::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of workaround_bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_launcher_page {
```

Comment:
As mention in the past, let's use the names on the screen instead of cook ours, it is always less confusing (ie what does it mean a launcher for each person).
`get_control_center_page`